### ISSUE TITLE
Error in customer scenario notebook

### ISSUE THREAD

**Issue by vsyrgkanis** [microsoft/EconML#437]
In cell [16] of the multi-investment attribution notebook:
https://github.com/microsoft/EconML/blob/7005b242531669854f9e0b6b944b590b9b4fdf66/notebooks/CustomerScenarios/Case%20Study%20-%20Multi-investment%20Attribution%20at%20A%20Software%20Company.ipynb

I believe it should be `X.shape[0]` in both places.

**Comment by vsyrgkanis** [comment]
Also I think in the end where you evaluate the ROI of each policy ,we should be using the true ROI, not the ROI estimated by the model, if we want to understand how we really performed.

So maybe you add another cell that displays the same comparison, but now with ground truth, something like:
```python
true_roi = np.zeros((X.shape[0], 4))
true_roi[:, 1:3] = TE_fn(X)
true_roi[:, 3] = true_roi[:, 1:3].sum(axis=1)
true_roi[:, 1:] -= cost_fn(multi_data)
```
and report the numbers in the table with the true_roi

**Comment by vsyrgkanis** [comment]
Addressed by #441.
